src/model/consent/consent.ts

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
```

Comment:
why this line?

Reply:
@lewisdaly suggested adding this because we haven't implemented BDD tests for models. These will be addressed in future tickets.

Reply:
So is there another workaround for now?

Reply:
I think it's fine we just need to also have a comment here that explains why we are ignoring it _for now_, that references a ticket for implementing the BDD tests for end to end consents creation/updating

Reply:
Addressed in a recent commit.